content/articles/faq-ssl-certificates.markdown

```diff
@@ -0,0 +1,79 @@
+---
+title: SSL Certificates Frequently Asked Questions
+excerpt: This is a collection of frequently asked questions about the SSL certificates offered by DNSimple.
+categories:
+- SSL Certificates
+---
+
+# SSL Certificate FAQ
+
+### Table of Contents {#toc}
+
+* TOC
+{:toc}
+
+---
+
+## Prices and Subscription
+
+<div class="section-faq" markdown="1">
+1.  #### What is the price of an SSL certificate?
+
+    The price depends on the certificate type. To learn more visit the [SSL Certificates](/articles/ssl-certificates/) article.
+
+1.  #### I want to purchase an SSL certificate. What is the DNSimple subscription?
+
+    DNSimple is a subscription-based service. In order to access our domain services, you need an active subscription to our platform.
+
+    The subscription gives you access to both the included platform services (such as the API access and the DNS hosting), and the paid products (such as the domain registration, SSL certificates, email forwarding, etc.).
+
+1.  #### I want to purchase an SSL certificate. Do I need a subscription?
+
+    Yes. As explained in our previous FAQ, the SSL certificates are one of our paid products, provided s part of our subscription services.
+
+    In order to purchase an SSL certificate you need an account active subscription at the time of the purchase. You are not required to have an active subscription to use or install the certificate on your domain/server, however please note that if you disable the subscription you will not receive any additional service for the domain or the certificate.
+
+    You will also need a valid, active subscription at the time you will need to renew the certificate, or if you want to reissue the certificate.
```

Comment:
I would say "You will also need a valid, active subscription when you renew the certificate..."